code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -488,8 +488,8 @@
 		environment = M.loc.return_air()
 	if(environment)
 		gases = environment.gases
-		if(gases[/datum/gas/plasma] && gases[/datum/gas/plasma][MOLES] > gases[/datum/gas/plasma][GAS_META][META_GAS_MOLES_VISIBLE]) //if there's enough plasma in the air to see
-			. += power * 0.5
+		if(gases[/datum/gas/plasma])
+			. += power * min(0.5, gases[/datum/gas/plasma][MOLES] * 1.1)
```

Comment:
What's the `1.1` for, is it a modifier or something? Could be a define

Reply:
Yeah it's the modifier for the healing rate. Basically it changes how much it heals by x1.1 based on how much plasma there is